graphql/subcourse/mutations.ts

```diff
@@ -80,6 +80,17 @@ export class MutateSubcourseResolver {
     ): Promise<GraphQLModel.Subcourse> {
         const course = await getCourse(courseId);
         await hasAccess(context, 'Course', course);
+        const courseInstructorAssociation = await prisma.course_instructors_student.findFirst({
+            where: {
+                courseId: courseId,
+                studentId: studentId,
```

Comment:
Ah, this for sure is wrong, we cannot trust our users to pass in their id. Please do

```
const student = await getSessionStudent(context, studentId);

... studentId: student.id
```